### PR TITLE
Merge master to nightly

### PR DESCRIPTION
**Pull request by mthalman**
Related to https://github.com/dotnet/dotnet-docker/issues/2583

### CODE REVIEW

File: manifest.versions.json [hunk 1]
```diff
@@ -32,18 +32,17 @@
...
     "dotnet|6.0|product-version": "6.0.0-preview.1",
 
     "monitor|5.0|build-version": "5.0.0-preview.3.21105.1",
     "monitor|5.0|product-version": "5.0.0-preview.3",
     "monitor|5.0|sha": "d01212f1a8e55341288e4840df5435ed9692df72d4f209a8bf3aca835748e376c43e2591028c3e55637ecb87e1c9dbaa719951753881c80c46b80f22faa96186",
 
-    "lzma|2.1|sha": "c4c81218e250242410bc4fe9039fc7f39dc5603febbe04605d6b9abc21c45273b79f12122bab973d49eacd4eefe38b2bc38a6f6cd30260e5613cabfb36e82a62",
+    "lzma|2.1|sha": "5604734ce516940fc37759c82aa6e9f26bb7391e2d6c2881ee0352ddcfff6f2c61db18fbed4bb03fcb81d77a6c1dbfdec0fea475226cee9715c9f2db28fa2ab9",
 
-    "powershell|3.1|build-version": "7.0.4",
```
Comment (MichaelSimons) [inline manifest.versions.json:46]: Why was this removed?

Comment (mthalman) [reply manifest.versions.json:46]: Too heavy on the delete key?  Restored it back.

**Review by MichaelSimons — APPROVED** [review]
